IApiCallback.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace BudgetSaverApp
 {
-    public interface IHasAPI
+    public interface IApiCallback
```

Comment:
it's still a really abstract name, but at least describes what the interface is for. Nonetheless, try to use domain language in your own code (transaction, possession, statistics, etc.), as those will be easier to read after N months, compared to technical-only names:)